README.md

```diff
@@ -147,27 +147,6 @@ The documentation is available here: [documentation](https://pypsa-earth.readthe
 <!-- readme: collaborators,contributors -start -->
 <table>
 <tr>
-    <td align="center">
```

Comment:
I've noticed this also last time.
Not sure if the cherry-picking intended to include this too

Reply:
Agree that it is annoying :) However, a contributor currently doesn't have an option to avoid fetching this git-creativity. I may look into settings of git actions if you feel it's necessary